src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication-context.xml

```diff
@@ -18,6 +15,21 @@
       <property name="defaultAdministratorUserNameList">
          <value>${token.authentication.defaultAdministratorUserNames}</value>
       </property>
+      <property name="active">
+         <value>${identity-service.authentication.enabled}</value>
+      </property>
+      <property name="authenticatorAuthzClient">
+         <ref bean="authenticatorAuthzClient"/>
+      </property>
+   </bean>
+
+   <bean name="authenticatorAuthzClient" class="org.alfresco.repo.security.authentication.identityservice.AuthenticatorAuthzClientFactoryBean">
+      <property name="identityServiceConfig">
+         <ref bean="identityServiceConfig" />
+      </property>
+      <property name="disabled">

```

Comment:
It is minor, but I think it is better to have a usual "enabled" flag as opposing to "disabled" flag.